kolibri/plugins/learn/assets/src/composables/useLearningActivities.js

```diff
@@ -93,8 +92,11 @@ export default function useLearningActivities(contentNode) {
     return (
       contentNode &&
       contentNode.duration &&
-      lodashGet(contentNode, ['options', 'completion_criteria', 'model']) ===
-        CompletionCriteria.APPROX_TIME
+      !(contentNode.kind == ContentNodeKinds.AUDIO || contentNode.kind == ContentNodeKinds.VIDEO) &&
```

Comment:
This might be more readable as:
```
contentNode.kind !== ContentNodeKinds.AUDIO && contentNode.kind !== ContentNodeKinds.VIDEO &&
```
and also means we are using a `===` equivalent rather than `==`.

Reply:
good idea, will update